### PR TITLE
minor changes while inspecting cli

### PR DESCRIPTION
**Pull request by memsharded**
Changelog: Omit
Docs: Omit

Just some minor changes while reading trying to refresh my understanding of the new CLI code.

### CODE REVIEW

File: conans/cli/output.py [hunk 1]
```diff
@@ -153,3 +153,8 @@ def _init_colors():
                 # TODO: check if colorama checks for stripping colors are enough
                 colorama.init()
             return True
+
+
+def cli_out_write(data, fg=None, bg=None, endline="\n"):
+    data = "{}{}{}{}{}".format(fg or '', bg or '', data, Style.RESET_ALL, endline)
+    sys.stderr.write(data)
```
Comment (memsharded) [inline conans/cli/output.py]: Not sure about the intention of this, if ``conan help`` output for example should go to stderr or to stdout? Is in this case the output considered the cli printing?

Comment (tapia) [reply conans/cli/output.py]: I think this should be `sys.stdout`, not `stderr`

Comment (czoido) [reply conans/cli/output.py]: Yes, this one should go to `sys.stdout ` this is the one that is used in the formatters of the commands and all the output should go to `stdout`.

Comment (memsharded) [reply conans/cli/output.py]: Fixed

**Review by czoido — APPROVED** [review]

**Pull request merged**